Fabric.Authorization.AccessControl/src/app/modules/access-control/member/member.component.html

```diff
@@ -3,12 +3,12 @@ <h3 class="h3">
     <span>Name (as it appears in directory)</span>
   </h3>
   <div hcInputIcon>
-    <input hc-input class="hc-input" (keyup)="searchText.next($event.target.value)">
+    <input hc-input class="hc-input" (keyup)="searchTextSubject.next($event.target.value)" [(ngModel)]="subjectId" [readOnly]="editMode">
```

Comment:
Would it better to not have it be an input, perhaps a title or label instead when editing?  The presence of a disabled input box can make the user think they either don't have permission to the box or that they need to do something to enable it.  As you will never have it enabled for an existing user record, IMHO it makes sense to not have an input box at all when `editMode`

Reply:
TODO: get feedback from others on what we should do about this